docs/dietpi_tools.md

```diff
@@ -325,6 +325,19 @@ It is one of the core tools, enabling you to install or uninstall one or more [*
     It converts a text file into morse code.  
     Run `dietpi-morsecode`.
 
+=== "Useful DietPi global commands (apt etc.)"
+
+    There are a couple of general commands which have a shortcut in DietPi:
+
+    - `cpu` - Print CPU clocks, temperature and governor info
+    - `G_TREESIZE` - Shows current directory/file sizes, recursive
+    - `G_AGI` - `apt-get install`
+    - `G_AGP` - `apt-get purge`
+    - `G_AGA` - `apt-get autoremove --purge`
+    - `G_AGUP` - `apt-get update`
+    - `G_AGUG` - `apt-get upgrade`
+    - `G_AGDUG` - `apt-get dist-upgrade`
```

Comment:
Probably we should add the info that these are error-handled, otherwise non-interactive wrappers, i.e. ideal for scripts, where follow-up commands depend on their success and you might want to log the output. But I would not recommend to use them manually in console, where the interactive y/n prompt and percentage processing outputs are nice to have while the G_EXEC error handler prompt and output are overkill 😄.

What could be added:
- `G_OBTAIN_CPU_TEMP` print integer format CPU temp in degree Celsius (for monitoring probably better than full `cpu` output.
- `G_OBTAIN_CPU_USAGE` CPU usage in percent in [x]y.z format.

Reply:
Ok, added the two points.

@MichaIng: What shall I add?  
Would you generate a textual proposal to insert.

Reply:
E.g. moving the the G_AG* commands to the end with a leading sentence:
```
The following commands are non-interactive, but error-handled wrappers for `apt-get` commands, useful for scripts which shall run non-interactive but allow interactive repeat and solution attempts.
```

Reply:
OK, text added.